### PR TITLE
access to websocket broadcaster

### PR DESCRIPTION
**Pull request by jacobrosenthal**
Im sending out a broadcaster object holding the ws::sender in order to not leak ws stuff which seems to work, but for now Im only accepting vec u8 for send, instead of From Message Any thoughts on how to implement that from without reexporting Message?

### CODE REVIEW

**Comment by jacobrosenthal** [comment]
[clabot:check]

**Comment by jacobrosenthal** [comment]
Well. That seemed to work just fine. Thoughts on the implementation?

File: ws/src/server.rs [hunk 1]
```diff
@@ -16,138 +16,181 @@ use crate::session;
 
 /// `WebSockets` server implementation.
 pub struct Server {
-	addr: SocketAddr,
-	handle: Option<thread::JoinHandle<Result<()>>>,
-	executor: Arc<Mutex<Option<Executor>>>,
-	broadcaster: ws::Sender,
+    addr: SocketAddr,
```
Comment (tomusdrw) [inline ws/src/server.rs]: We use tabs across the codebase please re-format or wait for #406

**Comment by tomusdrw** [comment]
Awesome, thanks a lot!

**Pull request merged**